src/AccessToken.php

```diff
@@ -167,8 +168,12 @@ private function determineAlg(array $certs)
      *                              the JWT.
      * @return array|bool the token payload, if successful, or false if not.
      */
-    private function verifyEs256($token, array $certs, $audience = null)
-    {
+    private function verifyEs256(
+        $token,
+        array $certs,
+        $audience = null,
+        $throwException = false
```

Comment:
Wouldn't it be easier to just allow the private methods to throw exceptions instead of `catch { if ($throwException) {`, bubble up to verify() and be suppressed or thrown there?

Reply:
This will look way cleaner, good call